ops/model.py

```diff
@@ -2744,7 +2744,6 @@ def get_notices(
         user_id: Optional[int] = None,
         types: Optional[Iterable[Union[pebble.NoticeType, str]]] = None,
         keys: Optional[Iterable[str]] = None,
-        after: Optional[datetime.datetime] = None,
```

Comment:
Thinking about it, you could just have some opaque `Notice.id` and then take `after=notice.id`. That field could be taken from last_repeated but is intentionally left opaque.